CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,7 @@
 - [Netbox] Support Netbox 4 (Thanks [@ChrisMcNichol](https://github.com/ChrisMcNichol)!) (PR [#7735](https://github.com/vatesfr/xen-orchestra/pull/7735))
 - [Migration] Disable CBT when needed during a disk/VM migration (PR [#7756](https://github.com/vatesfr/xen-orchestra/pull/7756))
 - [Disks] Show and edit the use of CBT (Change Block Tracking) in disks (PR [#7732](https://github.com/vatesfr/xen-orchestra/pull/7732))
+- [REST API] Added an endpoint for _rolling pool reboot_ (PR [#7761](https://github.com/vatesfr/xen-orchestra/pull/7761))
```

Comment:
Similar to what I did for RPU:

- capitalize feature name
- explicit the new endpoint

```suggestion
- [REST API] _Rolling Pool Reboot_ action available `pools/<uuid>/actions/rolling_reboot` (PR [#7761](https://github.com/vatesfr/xen-orchestra/pull/7761))
```